experimenter/tests/integration/nimbus/android/ping_server/ping_server.py

```diff
@@ -0,0 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
```

Comment:
Can we add a module docstring explaining what this is? I think its a dummy telemetry ingestion service?

Reply:
Correct, I can do that.